app/src/main/java/org/oppia/app/topic/review/TopicReviewViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package org.oppia.app.topic.review
+
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.domain.topic.TopicController
+import javax.inject.Inject
+
+/**
+ * [ViewModel] for showing a list of topic review-skills.
+ */
+@FragmentScope
+class TopicReviewViewModel @Inject constructor(fragment: Fragment) : ViewModel() {}
```

Comment:
Prefer not injecting ``Fragment`` if it's unused.

Reply:
Removed unused fragment.